lib/tasks/rebuild_ucf.rake

```diff
@@ -0,0 +1,21 @@
+namespace :freereg do
+
+  desc "Rebuild UCF and deploy"
+  task :rebuild_ucf => [:environment] do
```

Comment:
Let's rename this task to "find_bad_ucf", since a second task will consume the ID files and do the actual rebuild.